src/ansys/simai/core/data/projects.py

```diff
@@ -255,3 +255,21 @@ def delete(self, project: Identifiable[Project]) -> None:
             project: ID or :class:`model <Project>` of the project.
         """
         self._client._api.delete_project(get_id_from_identifiable(project))
+
+    def cancel_build(self, project: Identifiable[Project]):
+        """Cancel an ongoing build if one exists.
+
+        Args:
+            project: ID or :class:`model <Project>` of the project.
+
+        Returns:
+            ``True`` if ``is_being_trained=False`` in the JSON response.
```

Comment:
The end user doesn't care about "the JSON response".

Also in this case it seems like `is_being_trained=True` could never happen ? These cases are possible:
* project was training a model, we cancelled the training ->  `is_being_trained=False`
* project was not training a model -> `is_being_trained=False`
* an error happened -> exception is raised, `cancel_build` never returns